free-1.md

```diff
@@ -10,41 +10,63 @@ _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
 
+I don't know if "need" is the right word, but this DSL would let people (programmers or otherwise) make text adventure games. The current experience involves using a general-purpose programming language, or a DSL or other development system that already exists for this purpose. For the most part using the systems designed for this purpose saves time and effort, specialized as they are. While they are reasonably effective, the ones I've seen are English-heavy, very C-like, or not a
+language. As I'll say below, the English-heavy one I have in mind actually seems pretty effective as it is, but C-like syntax can be prohibitive to non-programmers, and the non-language system is slower to use, as such things are. In any case, with the project I have in mind, users would have an option similar to the English-heavy language, but less English-heavy. If we've been on the right track in our class discussions, this could reduce user frustration over syntax
+errors.
+
 
 ### Why a language?
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
 
+A library could certainly do the same job, but a DSL would be more accessible to people who aren't that familiar with programming in general. Also, it would be nice to make the DSL look declarative even if semantically it isn't, for reasons I've discussed in various places in this response.
+
 
 ### Why you?
 _What excites you about this idea? How did you come up with it?_
 
+I really like games in general, and I think it's unfortunate that people don't really make text adventures anymore. I had fun making one in CS60, but using Prolog was... less than ideal. In some respects, though, even that was easier to work with than I sort of imagine using Java or C++ or even Python would be; I think the declarative style is very well suited to making text adventures.
+
 
 ### Domain
 _Describe the project's domain in five words._
 
+It makes text adventure games.
+
 
 ### Interface (syntax)
 _How might the user interact with the language? What does programming look 
 like? Why is this the right way to interact with the problem domain?_ 
 
+Generally speaking, the user would describe the layout of the world, what is in it and where, and what actions the player is allowed to take. I conjecture (based entirely upon my own intuition) that many people would find it easier to create a world by describing it and letting the computer do the rest rather than by telling the computer how to set everything up. I would want to ask as many people as possible whether this is true for them before actually committing to this
```

Comment:
What do you mean by describing the layout of the world? Physical layout? How do you plan on describing this layout? I would think that some sort of map would be interesting. Like if the user could draw an ascii map of their world.